api/Dockerfile

```diff
@@ -7,18 +7,18 @@ ENV MINICONDA_PACKAGE Miniconda3-4.3.11-Linux-x86_64.sh
 ENV PYT_VERSION python3.6
 
 RUN apt-get update &&\
-    apt-get -y install postgresql-client libpq-dev libgeos-c1 &&\
+    apt-get -y install tk-dev curl postgresql-client libpq-dev libgeos-c1 &&\
     apt-get clean
 
 ENV PATH=/miniconda/bin:${PATH}
 ENV PYTHONPATH=/miniconda/lib/${PYT_VERSION}/site-packages
 
-RUN curl -LO https://repo.continuum.io/miniconda/${MINICONDA_PACKAGE}  &&\
-    bash ${MINICONDA_PACKAGE} -p /miniconda -b &&\
-    rm ${MINICONDA_PACKAGE} &&\
-    conda update -y conda &&\
-    conda install -y pandas &&\
-    pip install uwsgi
+RUN curl -LO https://repo.continuum.io/miniconda/${MINICONDA_PACKAGE}
+RUN bash ${MINICONDA_PACKAGE} -p /miniconda -b
+RUN rm ${MINICONDA_PACKAGE}
+RUN conda update -y conda
+RUN conda install -y pandas
+RUN pip install uwsgi
```

Comment:
It is good practise to keep the number of commands/layers/containers to a minimum in order to avoid eating up unecessary disk space.  I would combine all these run commands back into a single command again.